### DISCUSSION TITLE
strapi io config not Log the socket id of any newly connected sockets.

### DISCUSSION THREAD

**Discussion by RAzaALy** [strapi-community/plugin-io#19] [Q&A]
### What issue are you experiencing?

`module.exports = ({ env }) => ({
 
    "io": {
      "enabled": true,
      "config": {
        "IOServerOptions" :{
          "cors": { "origin": "http://localhost:1337", "methods": ["GET"] },
        },
        "contentTypes": {
          "Puzzles": "*",
        },
        "events":[
          {
            "name": "connection",
            "handler": ({ strapi }, socket) => {
              strapi.log.debug(`[io] new conection with id ${socket.id}`);
            },
          },
        ]
      },
    },
  });`

### What version of the plugin are you using?

v1.1.0

### What strapi version are you using?

v4.1.3

**Comment by hrdunn** [comment]
Are you able to confirm that your client socket is actually connecting? Are you connecting from http://localhost:1337? or another port? Try adding your client's domain in the cors
```
"cors": { "origin": ["http://localhost:3000", "http://localhost:3001"], "methods": ["GET"] },
```

**Comment by RAzaALy** [comment]
No client socket is not connected yet and I provide the backend origin
http:// localhost:1337

**Comment by hrdunn** [comment]
if your client socket isn't built/connected yet, your connection event will not trigger as nothing is connected to log the socket ID. Your content type should also be the singular name of the model.
